query/search.go

```diff
@@ -88,15 +88,15 @@ func SetupV710Search() (*template.Template, error) {
 }
 
 // BuildSearchQuery creates an elastic search query from the provided search parameters
-func (sb *Builder) BuildSearchQuery(ctx context.Context, q, contentTypes, sort string, topics []string, limit, offset int) ([]byte, error) {
+func (sb *Builder) BuildSearchQuery(ctx context.Context, q, contentTypes, sort string, topics []string, limit, offset int, esVersion710 bool) ([]byte, error) {
 	reqParams := searchRequest{
 		Term:  q,
 		From:  offset,
 		Size:  limit,
 		Types: strings.Split(contentTypes, ","),
 		//Topic:            topics, // Todo: This needs to be reintroduced when migrating to ES 7.10
 		SortBy:           sort,
-		AggregationField: "_type",
+		AggregationField: "type",
```

Comment:
By changing this does it not result in an issue with the API interacting with ES 2.2?